docs/user/whats-new.asciidoc

```diff
@@ -1,16 +1,258 @@
+Query-based annotations also come with improved overflow support, additional style options, and a customizable tooltip.
+
+
+[role="screenshot"]
+image::images/highlights-vis-query-annotations.png[Query annotations from separate data views in Lens]
+
+[discrete]
+==== Turn table values into links
+
+Visual cues are important to dashboard users, especially when it comes to clickable values.
+With the Table visualization in *Lens*, you can now enable *Directly filter on click* to turn
+any column value into a link that filters the data, or drills down into the data.
+
+[role="screenshot"]
+image::images/highlights-vis-lens-table.png[Table visualization showing links in the top 5 values column, width="75%"]
+
+[discrete]
+==== Clone visualization layers
+
+When you create visualization layers in *Lens*, you don’t have to start from a blank slate.
+You can now duplicate layers, making it easy to adjust the filters and data options
+from a common starting point.
+
+[role="screenshot"]
+image::images/highlights-vis-duplicate-layer.png[Actions menu for visualizations open to shoe the Duplicate layer item, width="50%"]
+
+[discrete]
+==== Normalize by unit
+
+Need to use *Normalize by unit* for more than just date histogram visualizations in *Lens*?
+Now you can! Suppose you have a metric visualization that displays the count of the
+records field. With *Normalize by unit*, you can view a per minute calculation of the field.
+
+[role="screenshot"]
+image::images/highlights-vis-normalize.png[Menu item for viewing a per minute calculation of a field]
+
+[discrete]
+==== Display the current time
+
+Markers make it easy for anyone viewing your dashboard to quickly understand important
+events on visualizations. With the new *Show current time marker* axis option,
+you can add a red line to signify where “now” is on time series visualizations.
+
+[role="screenshot"]
+image::images/highlights-vis-time.png[Visualization showing a red line that indicates the current time, width="75%"]
+
+[discrete]
+==== Collapse by option
+
+Let’s say you’re creating a partition chart with sampled data, such as metrics,
+and want to break down the last values from each reporting dimension, such as hosts.
+You can now use the new *Collapse by* option to break down and summarize the
+result of each last value from hosts without displaying the detailed reporting dimension.
+
+[role="screenshot"]
+image::images/highlights-vis-collapse.png[Collapse by option for breaking down and summarizing the last value, width="75%"]
+
+[discrete]
+[[highlights-8.5-maps]]
+=== Maps &mdash; hide tooltips
+
+Tooltips display when you hover over a document or cluster in a map.
+Don't want to show them? Turn them off with
+the new *Show tooltips* toggle in *Layer settings*.
+
+[role="screenshot"]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt553f1d51649d508d/63445a7f2a8d0a11633dbb8f/highlights-maps.gif[Show tooltips control in Layer settings]
+
+
+[discrete]
```

Comment:
@lcawl can you please review the sections. on machine learning and cases?

Reply:
Yup, we might add another blurb, but this LGTM for now.